src/include/platform/internal/GenericConfigurationManagerImpl.ipp

```diff
@@ -867,7 +868,7 @@ CHIP_ERROR GenericConfigurationManagerImpl<ImplClass>::_ComputeProvisioningHash(
     using HashAlgo = chip::Crypto::Hash_SHA256_stream;
 
     HashAlgo hash;
-    uint8_t * dataBuf = NULL;
+    chip::Platform::ScopedMemoryBuffer dataBuf;
```

Comment:
This is inside `namespace chip`, right?  Why the prefix?

Reply:
Amazingly enough compiler complained about Platform being ambigous, so I used it as such.

Reply:
Specifically at least chip::Platform and chip::System::Platform exist and for code that has 'using' on both chip and system, we get conflicts.

We should not get conflicts if headers had a rule of 'no using inside headers' but the ported code does not enforce that so we are stuck until we clean up code. Would prefer that cleanup to be independent of this PR.